yarn.lock

```diff
@@ -0,0 +1,8 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
It is what it is: remove it from repo

Reply:
Me and @polluxx just had a discussion and decided to leave the lock file in the repository

Reply:
ok